TokenSale.sol

```diff
@@ -85,12 +91,24 @@ contract TokenSale is TokenSaleInterface, Token {
 
     function refund() noEther {
         if (now > closingTime
-            && !isFunded
-            && msg.sender.call.value(balances[msg.sender])()) // execute refund
+            && !isFunded)
```

Comment:
you can now put the if back in one line :), no longer exceeds the 80 column limit